config/confighttp/confighttp_test.go

```diff
@@ -235,6 +235,9 @@ func TestHttpReception(t *testing.T) {
 				_ = s.Serve(ln)
 			}()
 
+			// Wait for the servers to start
+			<-time.After(10 * time.Millisecond)
```

Comment:
Is this necessary? If so, would it be better to keep checking if the server is ready rather than an arbitrary wait? Also do we need similar for `TestReceiveOnUnixDomainSocket` above?

Reply:
This is for http. GRPC has on the client waitforready enabled so no need to do this hack.